ui/lib/kv/addon/components/page/secret/paths.hbs

```diff
@@ -5,6 +5,13 @@
 
 <KvPageHeader @breadcrumbs={{@breadcrumbs}} @pageTitle={{@path}}>
   <:tabLinks>
+    <li>
+      <LinkTo
+        @route="secret.index"
```

Comment:
Curious why the link here is `secret.index` but everywhere else it's just `secret`. Should we be consistent?

Reply:
Yeah - I think I made this change earlier on. Preference for one over the other? I like that `secret.index` is explicit but not sure we need to be. 🤔